lesson3/ToDoList2/app/src/main/java/erica/todolist2/MainActivity.java

```diff
@@ -16,20 +16,18 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);                                                     //gets the layout for the view from activity main
-        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);                                     //defines the toolbar
-        setSupportActionBar(toolbar);                                                               //makes the toolbar
+        setContentView(R.layout.activity_main);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
 
         final Fragment listFrag = new ListFragment();
-        replaceFragment(listFrag);                                                                  //calls the replaceFragment method defined below for the current view to be the list fragment
```

Comment:
This might need a comment, but your comment here is a bit too much. I don't need to know where the method is defined.  See my updated comment: it might be helpful for the reader here to know why you're replacing the fragment with a list fragment on your `onCreate`, so I made a comment explaining that the default fragment is the list fragment.